recipes/tinytest/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+import os
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+class TinyTestConan(ConanFile):
+    name = "tinytest"
```

Comment:
I tend to agree with @uilianries and would feel more comfortable with `jfalcou-tts`

`tinytest` is a stretch on our naming convention

`tinytestsystem` might be fine too

Reply:
Went for tinytestsystem